app/src/main/java/com/safetyheads/akademiaandroida/splashscreen/SplashScreenViewModel.kt

```diff
@@ -2,11 +2,16 @@ package com.safetyheads.akademiaandroida.splashscreen
 
 import androidx.lifecycle.ViewModel
 import androidx.lifecycle.viewModelScope
+import com.safetyheads.domain.usecases.GetConfigUseCase
 import kotlinx.coroutines.launch
 
-class SplashScreenViewModel(private val splashScreenUseCase: SplashScreenUseCase): ViewModel() {
+class SplashScreenViewModel(
+    private val splashScreenUseCase: SplashScreenUseCase,
+    private val getConfigUseCase: GetConfigUseCase
+    ): ViewModel() {
 
-    val delay = viewModelScope.launch {
+    val getConfig = viewModelScope.launch {
+        getConfigUseCase.invoke()
```

Comment:
You just call invoke() and not use te returned value. UseCase just create an object flow but nothing else. Please try to debug your code and check how it work.